### DISCUSSION TITLE
Improve handling of broken shortcuts (renamed/deleted categories)

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53966] [Mobile] [answered]
GitHub Mobile beta

I created a shortcut (done ask me how, I don't remember) a long time ago:
![Screenshot_20230428-092538.png](https://user-images.githubusercontent.com/2119212/235160221-d78c5b26-d7d3-4171-9282-92cc4ae61d9c.png)

At some point after that, GitHub reorganized the discussion categories.

When I tap that entry now, I get:
![Screenshot_20230428-092526.png](https://user-images.githubusercontent.com/2119212/235160424-952e8cae-41da-4a63-97f4-af647966c6ce.png)

Ideally if GitHub Mobile can know a shortcut is broken, the shortcut list should tell me that up front. 

Something in the flow should help me fix my shortcuts. 

Depending on where shortcuts are stored, it'd be helpful if the app could tell me the last time the shortcut worked. It's possible it's to a repository or organization that is temporarily not visible to me, or it's possible the category is permanently gone and knowing when I last was able to use the shortcut would help me determine where to search to decide between removing the shortcut and waiting* in the hopes it might recover later.

*Waiting might involve contacting an entity, organization, or GitHub in order to either restore general availability of the object or my ability to access the object.

**Comment by taki-on** [comment, marked as answer]
Thank you for the report. I guess the shortcut is about a discussion in the GitHub public feedback. The URL has slightly changed, and it will break the shortcuts referring to the URL. We apologize the inconvenience and we will see what we can do to prevent it in the future. Thanks!

**Reply by Don-bot576** [reply]
Understood
